src/components/media-player-controls.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, Slider } from 'antd';
+import { AudioMutedOutlined, CaretRightOutlined, PauseOutlined, SoundOutlined } from '@ant-design/icons';
+
+function formatMillisecondsAsDuration(milliseconds) {
```

Comment:
Now that we have the `media-utils` we could put this in there too and throw in a couple of tests.

Reply:
Good idea, thanks! It was put and tests were thrown :-)